tts-finetuning-cvtool.ipynb

```diff
@@ -0,0 +1,902 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Pre-Processing"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This step downloads audio to text file lists from NVIDIA for LJSpeech and generates the manifest files. If you use your own dataset, you have to generate three files: `ljs_audio_text_train_manifest.json`, `ljs_audio_text_val_manifest.json`, `ljs_audio_text_test_manifest.json` yourself. Those files correspond to your train / val / test split. For each text file, the number of rows should be equal to number of samples in this split and each row for a single speaker dataset should be like:\n",
+    "\n",
+    "```\n",
+    "{\"audio_filepath\": \"path_to_audio_file\", \"text\": \"text_of_the_audio\", \"duration\": duration_of_the_audio}\n",
+    "```\n",
+    "\n",
+    "In case of multi-speaker dataset\n",
+    "\n",
+    "```\n",
+    "{\"audio_filepath\": \"path_to_audio_file\", \"text\": \"text_of_the_audio\", \"duration\": duration_of_the_audio, \"speaker\": speaker_id}\n",
+    "```\n",
+    "\n",
+    "An example row is:\n",
+    "\n",
+    "```\n",
+    "{\"audio_filepath\": \"actressinhighlife_01_bowen_0001.flac\", \"text\": \"the pleasant season did my heart employ\", \"duration\": 2.4}\n",
+    "```\n",
+    "\n",
+    "We will now download the audio and the manifest files then convert them to the above format, also normalize the text. These steps for LJSpeech can be found in NeMo [`scripts/dataset_processing/tts/ljspeech/get_data.py`](https://github.com/NVIDIA/NeMo/blob/main/scripts/dataset_processing/tts/ljspeech/get_data.py). Be patient, this step is expected to take some time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! wget https://raw.githubusercontent.com/NVIDIA/NeMo/main/scripts/dataset_processing/tts/ljspeech/get_data.py\n",
+    "    \n",
+    "! python get_data.py --data-root {DATA_DIR}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "original_data_json = os.path.join(os.environ[\"DATA_DIR\"], \"LJSpeech-1.1/train_manifest.json\")\n",
+    "os.environ[\"original_data_json\"] = original_data_json"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's now download the data from the NVIDIA Custom Voice Recorder tool, and place the data in the `DATA_DIR`. Name the manifest file as `manifest.json`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "# Name of the untarred dataset from the NVIDIA Custom Voice Recorder.\n",
+    "finetune_data_name = FIX_ME\n",
```

Comment:
I think moving all the FIX_ME variables into the first code block of the notebook will cause confusion and might break the flow of the tutorial. For example:
`fastpitch_checkpoint = FIXME` this is used in generating mels for fine-tuning hifigan, the checkpoint is only obtained after performing fine-tuning FastPitch step.
`hifigan_checkpoint = FIXME` this checkpoint path is only obtained after fine-tuning HiFiGAN.
and so on.
Sure, we can move them up but we have to explain in the first block about all these variables. It might also make the user think how he/she can add the FIX_MEs in the first block.